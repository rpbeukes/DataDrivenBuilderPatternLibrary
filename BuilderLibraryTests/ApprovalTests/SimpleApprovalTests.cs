using ApprovalTests;
using ApprovalTests.Reporters;
using Data.Models;
using Newtonsoft.Json;
using System;
using System.Linq;
using Xunit;

namespace BuilderLibraryTests.ApprovalTests
{
    public class SimpleApprovalTests
    {
        [Fact]
        [UseReporter(typeof(DiffReporter))]
        public void ThisIsASimpleTests()
        {
            var address = new Address()
            {
                PostCode = "6000",
                StreetNumber = 12,
                Suburb = "Wallaby"
            };

            var jsonAddress = JsonConvert.SerializeObject(address, Formatting.Indented);

            Approvals.Verify(jsonAddress);
        }
    }
}
