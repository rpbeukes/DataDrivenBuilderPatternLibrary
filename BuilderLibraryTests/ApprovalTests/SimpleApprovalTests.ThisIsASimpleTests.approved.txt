{
  "StreetNumber": 12,
  "Suburb": "Wallaby",
  "PostCode": "6000"
}